habitat_sim/nav/greedy_geodesic_follower.py

```diff
@@ -65,7 +65,7 @@ def __attrs_post_init__(self):
     def _find_action(self, name):
         candidates = list(
             filter(
-                lambda v: v[1].name == name,
+                lambda v: name in v[1].name,
```

Comment:
This is a little dangerous as things are assumed to be deterministic in the follower, but #546 will subsume this, so fine.